### PR TITLE
Bug 2048186: Fix panic in finalizer handler

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: pkg/operator/finalizer.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (c *Controller) finalizeResources(o *imageregistryv1.Config) error {
 		if cr == nil {
 			// Skip using the cache here so we don't have as many
 			// retries due to slow cache updates
-			cr, err := client.Configs().Get(
```
Comment (dmage) [inline pkg/operator/finalizer.go:60]: The actual fix.

**Comment by xiuwang** [comment]
failed in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-image-registry-operator/745/pull-ci-openshift-cluster-image-registry-operator-master-e2e-aws-operator/1487172162581172224

**Comment by dmage** [comment]
@xiuwang this job doesn't contain the latest update, so it can be ignored. The current commit is e12d50c.

**Comment by dmage** [comment]
You can check commit history at https://prow.ci.openshift.org/pr-history/?org=openshift&repo=cluster-image-registry-operator&pr=745 - old commits are always broken, otherwise there wouldn't be need for changes

**Comment by dmage** [comment]
TestImageRegistryRemovedWithImages is another source of failures, unrelated to this PR.
/retest

**Review by ricardomaraschini — APPROVED** [review]
/lgtm

**Pull request merged**